### PR TITLE
more precise?

### PR DESCRIPTION
**Pull request by loldrup**

### CODE REVIEW

File: documentation/1.2/tour/inheritance.md [hunk 1]
```diff
@@ -301,8 +301,8 @@ an abstract superclass:
 Ceylon requires us to annotate abstract classes `abstract`, just like Java. 
 This annotation specifies that a class cannot be instantiated, and can define 
 abstract members. Like Java, Ceylon also requires us to annotate "abstract" 
-members that don't specify an implementation. However, in this case, the 
-required annotation is `formal`. The reason for having two different 
+members (i.e. members that don't specify an implementation. However, in this
+case, the required annotation is `formal`. The reason for having two different
```
Comment (jvasileff) [inline documentation/1.2/tour/inheritance.md]: The right paren is missing. Although another wording w/o parents might be "..., which are members that...".

Comment (jvasileff) [inline documentation/1.2/tour/inheritance.md]: "Parents", haha, iOS autocorrect

Comment (gavinking) [inline documentation/1.2/tour/inheritance.md]: I am not a fan of stuff like "(i.e. blah blah blah)". To me that's just a really lazy writing style.

**Comment by loldrup** [comment]
Challenge accepted :)

I choose to propose a rewrite the whole section of texts. The section was
mightily confusing to me when I read it, and I believe it has many
problems. My proposal doesn't even fix them all!

Proposal:

"""
Ceylon requires us to annotate abstract classes with the keyword
`abstract`, just like in Java. This annotation specifies that the given
class can't be instantiated, and also that it, in contrast to normal
classes, can define abstract members (temporary note: can it also define
non-abstract members?). Abstract members are members that don't specify an
implementation (alternatively: "abstract members are members for which you
*can't* specify an implementation in the abstract class itself").
Any members of an abstract class must also be declared to be 'abstract' in
some sense, although the required annotation in this case is usually the
word `formal` (temporary note: I write 'usually' because it says further
down that one type of members (nested classes) can actually be denoted
`abstract`, not `formal`). The reason for having two different annotations
for abstract classes and class members, as we'll see later, is that nested
classes may be either `abstract` or `formal`, and abstract nested classes
are a bit different to formal member classes: A formal member class may be
instantiated; an abstract class may not be.
'""

As far as I can tell, the text above introduces no additional structural
problems, compared to the old text, but at the same time, it doesn't
mitigate all of the problems of the old text. For instance, it says that
abstract members can't be implemented, but later on it is written that
formal members may be instantiated. How is it possible to instantiate
something that isn't implemented?

My suggestion is that someone knowledgeable of Ceylon checks my proposal
for factual errors, and update the documentation using that checked text.
Then we can take a second round of fine tuning afterwards. I can't propose
fine tunings as long as there are unclarified factual questions remaining
in my proposal.

Mvh. Jon Loldrup

**Comment by FroMage** [comment]
This PR has been closed as a result of moving Ceylon to the Eclipse Foundation, not as a judgement of its value. We are unfortunately unable to restore it ourselves, but if you feel this PR should be reopened, please rebase your work on the current master and create a new PR. Thanks, and sorry for the inconvenience.

**Pull request closed without merging**